APISpecification.yaml

```diff
@@ -530,6 +608,16 @@ components:
         - related_genomes_count
       additionalProperties: false
       type: object
+    ExampleObject:
+      type: object
+      properties:
+        id:
+          type: string
+          example: ENSG00000139618
+        type:
+          type: string
+          description: Member of a controlled vocabulary. Examples include "gene", "location", "variant"
```

Comment:
Notice that the current endpoint ([link](https://beta.ensembl.org/api/genomesearch/genome/info?genome_id=grch38)) has a `region` type. We have switched to calling it `location` on the client; and thus should probably use the term `location` in the api as well.